src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocpackage/InputJavadocPackageBadCls.java

```diff
@@ -1,9 +1,13 @@
+/*
+JavadocPackage
+allowLegacy = (default)false
+fileExtensions = (default)java
+
+
+*/
+
 package com.puppycrawl.tools.checkstyle.checks.javadoc.javadocpackage;      // violation
```

Comment:
maybe move violations that occur at first line before `package` declaration?
because there are checks like [PackageName](https://checkstyle.sourceforge.io/config_naming.html#PackageName), it will be hard to distinguish if violation is related to file or to package

@romani @rnveach ?

Reply:
we can skip such Check inputs for now and deal with them at the end or completely leave them on old format of test+Inputs they are not out primary target and they might not need BDD ideas, as violation is always on first line, just comment enforcemen/validation will be enough.

Reply:
Can we do it like this?
```
/* // violation
JavadocPackage
allowLegacy = (default)false
fileExtensions = (default)java


*/
```

Reply:
@shashwatj07 we will back to this PR, after we fix other issues and have general understanding, for now lets avoid checks that throw violations for whole file